README.md

```diff
@@ -361,6 +361,8 @@ add_filter(
 
 The **jwt_auth_authorization_header** allows you to modify the Authorization header key used to validating a token. Useful when the server already uses the 'Authorization' key for another auth method.
 
+The header is retrieved from `$_SERVER` which means headers are prefixed with `HTTP_` and converted to capitalized snake case. Example: The key for a custom `X-Auth` header is `HTTP_X_AUTH`.
```

Comment:
This is standard PHP behavior and not invented here.  Therefore I do not think we should add this to the documentation.

As a compromise, we could maybe set up an FAQ section or similar in the [Wiki of the GitHub project](https://github.com/usefulteam/jwt-auth/wiki), in which everyone can contribute their learnings?

Reply:
The idea here is that it is not immediately clear from the description how the custom header key is supposed to be passed to the filter. My first attempt was `X-Auth` then I tried `HTTP_X-Auth` and finally needed to take a closer look at the PHP docs to figure it out. I don't know how common that knowledge is so I wanted to provide a quick tip that can save some time.

But in general a Wiki would be nice.